spec.bs

```diff
@@ -2859,6 +2864,175 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a {{WindowOrWorkerGlobalScope/fetch()}} call using the
+`{adAuctionHeaders: true}` option, which causes the value of the [:Ad-Auction-Signals:] response
+header on that request to be used to provide signals in auctions running on the page whose
+seller matches the origin of the {{WindowOrWorkerGlobalScope/fetch()}} destination.
+
+<div algorithm="fetch capture adAuctionHeaders boolean patch">
+Modify the definition of a [=request=]: 
+
+A [=request=] has an associated boolean <dfn for=request>capture-ad-auction-headers</dfn>.
+Unless stated otherwise it is false.
+</div>
+
+<div algorithm="fetch capture adAuctionHeaders RequestInit patch">
+Modify [[FETCH]]'s {{RequestInit}} dictionary to add an adAuctionHeaders attribute:
+
+<pre class=idl>
+partial dictionary RequestInit {
+  boolean adAuctionHeaders;
+};
+</pre>
+</div>
+
+<div algorithm="fetch new request patch">
+The following step will be added to the <code><a constructor for="Request" lt="Request()">new Request (<var ignore>input</var>, <var ignore>init</var>)</a></code>
+constructor steps, before step "Set this's [=Request/request=] to |request|":
+
+1. If <var ignore=''>init</var>["{{RequestInit/adAuctionHeaders}}"] <a for=map>exists</a>, then set |request|'s
+  [=request/capture-ad-auction-headers=] to it.
+
+</div>
+
+<div algorithm="fetch Sec-Ad-Auction-Fetch patch">
+The following step will be added to the [=http-network-or-cache fetch=] algorithm, before step
+"Modify |httpRequest|'s [=request/header list=] per HTTP. ...":
+
+1. If [=request=]'s [=request/capture-ad-auction-headers=] is true, then
+  [=header list/set a structured field value=] given «[:Sec-Ad-Auction-Fetch:], the [=structured header/boolean=] `?1`» in

```

Comment:
Can you hard-wrap this after "structured" and before "header" (and then unwrap the short line before)

Reply:
The wrapping turned out a bit differently when unwrapping the line before (is it typical to wrap in the middle of a link?) -- now I wrap the second line before "in", since that's where I now meet the 100 char limit on the next line.

Let me know what you think -- also are there auto formatters for Bikeshed?

Reply:
> (is it typical to wrap in the middle of a link?)

It is in all WHATWG specs, which is what I go off of (and probably most WICG specs too?)

> Let me know what you think -- also are there auto formatters for Bikeshed?

Oh boy this is a dangerous question! We technically have two crappy options:
 - https://domenic.github.io/rewrapper, which is what I'd use for individual lines (the default setting of 100 columns is right)
 - https://github.com/domfarolino/specfmt — this isn't bad, but it doesn't yet work for numbered lists just yet unfortunately, so it might not be useful here.

Anyways, all looks well here so I'll resolve this! Thanks!